### PR TITLE
feat: home page

### PR DESCRIPTION
**Pull request by harshitadaswani**
- added home page
- linked font awesome stylesheets in category
- added utility classes needed in index.html
- added SVG needed for index page
- preview link: https://deploy-preview-2--awesome-clarke-46c76b.netlify.app/

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
good to merge

**Review by shaksham08 — APPROVED** [review]
Looks good to me

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            </div>
+            <div class="home-container p-s m-s">
+                <div class="action-color p-m m-m m-auto txt-xl fw-bold">Welcome to Crypto Quiz</div>
+                <label for="Name" class="p-m m-xs txt-m fw-semibold">Enter your Name:</label>
+                <div class="input-icon relative m-s">
+                    <i class="fas fa-user user-icon"></i>
+                    <div class="flex flex-column">
+                        <input type="text" name="Name" class="br-m p-s txt-s m-s txt-center border-none outline-transparent name fw-semibold">
+                        <a href="./screens/category.html">
+                            <input type="button" value="Play!" class="btn txt-cursor outline-transparent border-none fw-semibold">
```
Comment (JantuDeb) [inline index.html:27]: Any specific reason to use a input for button

**Review by JantuDeb — APPROVED** [review]
Codes look clean and structured

File: index.html (hunk 1 above)
Comment (harshitadaswani) [reply index.html:27]: I want to use submit feature but without javascript, it was refreshing on the click so changed it to the button type.

**Pull request merged**